internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -115,6 +116,45 @@ func SetupLogger(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionF
 	}, "setupLogger"
 }
 
+// switchConsole switches the active console to the specified tty.
+func switchConsole(ttyNumber int) error {
+	// redirect the kernel logs to tty1 instead of the currently used one,
+	// so that dashboard on tty2 does not get flooded with kernel logs
+	if err := redirectKernelLogs(1); err != nil {
+		return err
+	}
+
+	// we need a valid fd to any tty because ioctl requires it
+	tty0, err := os.OpenFile("/dev/tty0", os.O_RDWR, 0)
+	if err != nil {
+		return err
+	}
+
+	defer tty0.Close() //nolint: errcheck
+
+	if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, tty0.Fd(), 0x5606 /* VT_ACTIVATE */, uintptr(ttyNumber)); errno != 0 {
```

Comment:
maybe add this as a comment here https://www.kernel.org/doc/html/v5.18/fb/fbcon.html :shrug:

Reply:
This doesn't have anything to do with framebuffer, but I can add this: https://man7.org/linux/man-pages/man2/ioctl_console.2.html

Reply:
ahh, i though the VT_ACTIVATE was in the link , yeh man page would work

Reply:
they are not nicely documented, I added some links to the relevant parts of the kernel source code both here and in `redirectKernelLogs`.

Reply:
done